src/js/components/LoginPage/Login.scss

```diff
@@ -0,0 +1,127 @@
+$primary-color: #fff;
+
+.login{
+  background-color: #bbb;
+  min-height: 100vh;
+  color: $primary-color;
+  padding-bottom: 30px;
+  & span {
+    display: block;
+    overflow: hidden;
+    padding: 0 4px 0 6px
+  }
+}
+
+label {
+  float: left
+}
+
+input {
+  width: 100%
+}
+.form-input{
+  background-color: transparent;
+  border: 1px solid $primary-color;
```

Comment:
if the border and the input's colours are the same can you actually see the border?

Reply:
Yes I can as the ancestrial element has a dark background color so the frame of this div is white and it's text white but it's background dark.